src/pint/pintk/colormodes.py

```diff
@@ -5,6 +5,28 @@
 import pint.logging
 from loguru import logger as log
 
+named_colors = {
```

Comment:
Matplotlib has a list of named colors: https://matplotlib.org/stable/gallery/color/named_colors.html

Reply:
Yeah, that's a good point.  We probably shouldn't define our own.  If you do `import matplotlib.colors as mcolors` then `mcolors.cnames`, `mcolors.TABLEAU_COLORS`, and `mcolors.XKCD_COLORS` are all dictionaries just like you have defined.

The xkcd one is cute:  https://xkcd.com/color/rgb/

Reply:
Yes, I was trying to minimally change what was there.  There's also the question of an appropriate subset (usually I would use seaborn, but those aren't always nameable) but it wouldn't be too hard to identify the XKCD color most similar to the ones already chosen.